vendor/github.com/giantswarm/certificatetpr/custom_object.go

```diff
@@ -12,7 +14,7 @@ import (
 // specifications of the resource the Certificate operator is interested in.
 type CustomObject struct {
 	unversioned.TypeMeta `json:",inline"`
-	Metadata             api.ObjectMeta `json:"metadata"`
+	Metadata             api.ObjectMeta `json:"metadata" yaml:"metadata"`
```

Comment:
I messed up when I applied the "decoding magic". The yaml encoding was in spec but not in CustomObject and List. I think this was making the decoder sad.

Its also possible we need to remove the yaml support entirely. Testing that next.